qa/integration-tests/src/test/java/io/zeebe/broker/it/health/DiskSpaceRecoveryITTest.java

```diff
@@ -0,0 +1,138 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.0. You may not use this file
+ * except in compliance with the Zeebe Community License 1.0.
+ */
+package io.zeebe.broker.it.health;
+
+import static org.assertj.core.api.Assertions.assertThatCode;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.awaitility.Awaitility.await;
+
+import com.github.dockerjava.api.model.Bind;
+import com.github.dockerjava.api.model.Volume;
+import io.zeebe.client.ZeebeClient;
+import io.zeebe.containers.ZeebeBrokerContainer;
+import java.time.Duration;
+import java.util.Map;
+import java.util.concurrent.TimeUnit;
+import org.elasticsearch.client.RestClient;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.testcontainers.containers.Network;
+import org.testcontainers.elasticsearch.ElasticsearchContainer;
+
+public class DiskSpaceRecoveryITTest {
+  static ZeebeBrokerContainer zeebeBroker;
+  private static final Logger LOG = LoggerFactory.getLogger("TEST");
+  private static final String VOLUME_NAME = "tmpdata1";
+  private static String containerIPAddress;
+  private static Integer apiPort;
+  private static ElasticsearchContainer elastic;
+  private ZeebeClient client;
+
+  @BeforeClass
+  public static void setUpClass() {
+    final var elasticHostInternal = "http://elastic:9200";
+    zeebeBroker = createZeebe(elasticHostInternal);
+    final var network = zeebeBroker.getNetwork();
+    elastic = createElastic(network);
+    zeebeBroker.start();
+
+    apiPort = zeebeBroker.getMappedPort(26500);
+    containerIPAddress = zeebeBroker.getContainerIpAddress();
+  }
+
+  private static ZeebeBrokerContainer createZeebe(final String elasticHost) {
+    zeebeBroker =
+        new ZeebeBrokerContainer("current-test")
+            .withEnv(
+                "ZEEBE_BROKER_EXPORTERS_ELASTICSEARCH_CLASSNAME",
+                "io.zeebe.exporter.ElasticsearchExporter")
+            .withEnv("ZEEBE_BROKER_EXPORTERS_ELASTICSEARCH_ARGS_URL", elasticHost)
+            .withEnv("ZEEBE_BROKER_EXPORTERS_ELASTICSEARCH_ARGS_BULK_DELAY", "1")
+            .withEnv("ZEEBE_BROKER_EXPORTERS_ELASTICSEARCH_ARGS_BULK_SIZE", "1")
+            .withEnv("ZEEBE_BROKER_DATA_SNAPSHOTPERIOD", "1m")
```

Comment:
Ok this test will take at least 1 minute right - wouldn't be nice to have something like https://github.com/zeebe-io/enhancements/pull/5

Reply:
Is there something else we could do at all? Just wondering